### PR TITLE
Fix Multi Attribution breaks mobile app

### PR DESCRIPTION
**Pull request by tsteur**

### CODE REVIEW

File: tiapp.xml [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
+    <sdk-version>12.6.1.GA</sdk-version>
     <id>org.piwik.mobile2</id>
     <name>Matomo Mobile 2</name>
-    <version>2.5.2</version>
+    <version>2.5.5</version>
     <publisher>thomassteur</publisher>
     <url>https://matomo.org/mobile</url>
     <description>Need your analytics on the go? Use our free Matomo Mobile (formerly Piwik Mobile) app to check your stats from anywhere and show them to colleagues in meetings. Matomo Mobile offers all the same reports as the Matomo (Piwik) desktop interface, and displays your visitor data in real time!</description>
-    <copyright>2023 by matomo</copyright>
+    <copyright>2010 by matomo</copyright>
```
Comment (michalkleiner) [inline tiapp.xml:14]: This probably should stay at 2023 or even 2024?

Comment (michalkleiner) [reply tiapp.xml:14]: Or is it intentionally going back to when Matomo started?

Comment (tsteur) [reply tiapp.xml:14]: it's intentional

**Comment by tsteur** [comment]
Not using latest titanium sdk 12.6.1. Various clicks, especially in side menus aren't working anymore. Couldn't figure out what's happening. Changelog not helpful either, may be a regression. Upgrading to 12.5.1 instead

**Comment by tsteur** [comment]
Seems to work on phones. Need to test it on a tablet once I can

**Pull request merged**